hackathon/views.py

```diff
@@ -70,6 +70,55 @@ def create_hackathon(request):
     pass
 
 
+@login_required
+def update_hackathon(request, hackathon_id):
+    """ Allow users to edit hackathon event """
+
+    # Redirect user if they are not admin
+    if not request.user.is_superuser:
+        return redirect("hackathon:hackathon-list")
+
+    hackathon = get_object_or_404(Hackathon, pk=hackathon_id)
+
+    if request.method == 'GET':
+
```

Comment:
Please remove the unneeded blank line

Reply:
Done, this will be visible when I push the change to the existing PR.